mne/tests/test_annotations.py

```diff
@@ -853,6 +853,41 @@ def test_read_annotation_txt_orig_time(
     assert_array_equal(annot.duration, [42., 48])
     assert_array_equal(annot.description, ['AA', 'BB'])
 
+@pytest.fixture(scope='session')
```

Comment:
Run `make flake` locally. It will complain (at least) about not having two newlines between function defs